openmrs-19/src/main/java/org/motechproject/openmrs19/service/OpenMRSUserService.java

```diff
@@ -22,67 +22,82 @@
     String PASSWORD_KEY = "password";
```

Comment:
Can you update javadoc for constants in this service ? Probably the constants should be removed, however it is task for another ticket. So just update javadocs